src/TrackerDevice.cpp

```diff
@@ -59,6 +59,26 @@ void SlimeVRDriver::TrackerDevice::PositionMessage(messages::Position &position)
     pose.qRotation.y = position.qy();
     pose.qRotation.z = position.qz();
 
+    auto current_universe = GetDriver()->GetCurrentUniverse();
+    if (current_universe.has_value()) {
+        auto trans = current_universe.value();
+        // pose.vecPosition[0] -= trans.translation.v[0];
+        // pose.vecPosition[1] -= trans.translation.v[1];
+        // pose.vecPosition[2] -= trans.translation.v[2];
+
+        // TODO: set this once, somewhere?
+        pose.vecWorldFromDriverTranslation[0] = -trans.translation.v[0];
+        pose.vecWorldFromDriverTranslation[1] = -trans.translation.v[1];
+        pose.vecWorldFromDriverTranslation[2] = -trans.translation.v[2];
+
+        pose.qWorldFromDriverRotation.w = cos(trans.yaw / 2);
+        pose.qWorldFromDriverRotation.x = 0;
+        pose.qWorldFromDriverRotation.y = sin(trans.yaw / 2);
+        pose.qWorldFromDriverRotation.z = 0;
```

Comment:
This is something that doesn't actually need to be checked/set every single time we send the pose, we could just do something to update it every time we detect a change instead.